### PR TITLE
Fixes #8065: ACL Posix on Git repos

### PR DESCRIPTION
**Pull request by amousset**
https://www.rudder-project.org/redmine/issues/8065

### CODE REVIEW

File: rudder-agent/debian/postinst [hunk 1]
```diff
@@ -163,6 +163,13 @@ case "$1" in
       chmod 644 ${TMP_CRON}
     fi
 
+    # Try to remove POSIX ACL if present, only during the first install
+    # http://www.rudder-project.org/redmine/issues/8065
+    if [ ${CFRUDDER_FIRST_INSTALL} -eq 1 ]
+    then
+      setfacl -R -k /var/rudder/ >/dev/null 2>/dev/null &
```
Comment (peckpeck) [inline rudder-agent/debian/postinst]: Why do you add the /dev/null redirect ?

**Comment by amousset** [comment]
PR updated

File: rudder-agent/debian/postinst [hunk 2]
```diff
@@ -163,6 +163,13 @@ case "$1" in
       chmod 644 ${TMP_CRON}
     fi
 
+    # Try to remove POSIX ACL if present, only during the first install
+    # http://www.rudder-project.org/redmine/issues/8065
+    if [ ${CFRUDDER_FIRST_INSTALL} -eq 1 ]
+    then
+      setfacl -R -k /var/rudder/ >/dev/null 2>/dev/null
```
Comment (peckpeck) [inline rudder-agent/debian/postinst]: Same question here

**Comment by amousset** [comment]
PR updated

**Comment by peckpeck** [comment]
OK, merging

**Pull request merged**